src/libs/types-rs/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "kata-types"
+version = "0.1.0"
+description = "Constants and data types shared by Kata Containers components"
+keywords = ["kata"]
```

Comment:
Maybe you could add some more keywords like "container" and "virtualization" here?

Reply:
added "containers", "runtime"